plutus-tx-plugin/src/Language/PlutusTx/Plugin.hs

```diff
@@ -275,12 +275,9 @@ compileCoreExpr (opts, famEnvs) locStr codeTy origE = do
             in if poDeferErrors opts
             -- this will blow up at runtime
             then do
-                defUni <- GHC.lookupTyCon =<< thNameToGhcNameOrFail ''PLC.DefaultUni
-                defFun <- GHC.lookupTyCon =<< thNameToGhcNameOrFail ''()
```

Comment:
WHOOOPS. I wrongly used `()` instead of `PLC.DefaultFun`. How come that did not trigger any error? @michaelpj

Reply:
I think this is because we run *after* the typechecker. So you can make ill-typed things. In this case it doesn't matter because it's phantom. Perhaps it would have failed with core lint on.